CHANGELOG.md

```diff
@@ -10,6 +10,9 @@
 * Set default 1G heap for 64-bit (Windows)
 * Updates for OPTUS D1 transponder changes to DVB-S2
 * Fix: Schedules Direct EPG grabber failed to finish updating some satellite-based lineups
+* Added the ability to use a Plugin for format detection of media files instead of built in ffmpeg.
+* Added 2160p as a Pretty resolution to VideoFormat
+* Added HEVC as a supported media format

```

Comment:
You didn't include the new HEVC constant.

Reply:
Good Catch.  I committed that change to the Branch/Pull request.  It is in MediaFormat class